app/models/previously_published_item.rb

```diff
@@ -27,6 +27,14 @@ def first_published_at
     previously_published_item.first_published_at
   end
 
+  def set_last_edited_at?
```

Comment:
I don't really understand the name for this method though I can see there's a `set_first_published_at?` method so I can see it's consistent.
My understanding is that it is used to determine whether the previous item has a `last_edited_at` timestamp? In which case `has_last_edited_at?` would make more sense as `set_` prefixed methods are usually mutators*.

`* The biased opinion of an ex-Java developer.`